packages/fuchsia_remote_debug_protocol/examples/list_vms_and_flutter_views.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
```

Comment:
Actually, we already have an `examples` directory at the root of the repository meant to demo all kinds of stuff. This should probably go there.

Reply:
That directory contains demos of flutter itself (it even says that you should run `flutter run` inside example directories in the readme), so I don't think that would be appropriate.

Reply:
Ah, got it.